main.go

```diff
@@ -200,6 +198,59 @@ func getTrailheadID(w http.ResponseWriter, userAlias string) string {
 	return userAlias
 }
 
+func doTrailheadAuraCallout(apexAction string, pageURI string) trailhead.Data {
```

Comment:
Can you add a description for this method starting with the name of the method? In Golang, you'd do:

```
// doTrailheadAuraCallout calls out to trailhead .... <rest of description>
```